docker-compose.yml

```diff
@@ -94,6 +97,7 @@ services:
       dockerfile: ./docker/indexer/Dockerfile
       args:
         URL: "https://github.com/algorand/indexer"
+        # Can we follow up on this now?
```

Comment:
Yes, but it would still be pinned to the commit mapping to the 2.3.x release. IIRC this is the release where we started importing go-algorand.